scanners/boostsecurityio/npm-audit/module.yaml

```diff
@@ -2,7 +2,7 @@ api_version: 1.0
 
 
 id: boostsecurityio/npm-audit
-name: BoostSecurity npm-audit
+name: Npm-audit
```

Comment:
Seems linke a tool from `npm` which is lower case, should it change?
https://docs.npmjs.com/cli/v9/commands/npm-audit

Reply:
fixed